src/action-ros-ci.ts

```diff
@@ -130,9 +130,7 @@ async function run() {
 			? sourceRosBinaryInstallation.split(RegExp("\\s"))
 			: [];
 
-		const vcsRepoFileUrlListAsString = core.getInput("vcs-repo-file-url", {
-			required: true
-		});
+		const vcsRepoFileUrlListAsString = core.getInput("vcs-repo-file-url");
```

Comment:
```suggestion
		const vcsRepoFileUrlListAsString = core.getInput("vcs-repo-file-url") || "";
```

This way, `undefined` will be replaced by "" properly.

Reply:
Done.